src/layouts/AboutLayout.js

```diff
@@ -0,0 +1,338 @@
+  return info.map(i => {
+    return <InfoBubble data={i} key={i.title}></InfoBubble>;
+  });
+}
+
+function renderMemberInfo(members) {
+  return members.map(m => {
+    return <Member data={m} key={m.name}></Member>;
+  });
+}
+
+export default function about() {
+  return (
+    <Container>
+      <AboutImageContainer>
+        <HeadlineContainer>
+          <Headline>
+            <HeadlineText>CLIMATE CONNECT</HeadlineText>
+          </Headline>
+        </HeadlineContainer>
+      </AboutImageContainer>
+      <h2>
+        We are an international team of volunteers building a non-profit climate action webplatform
+      </h2>
+      <h1>Our goal is to help you fight climate change most effectively</h1>
+      <BubbleGrid>{renderBubbles(info)}</BubbleGrid>
+      <h1>Find out more</h1>
+      <div>
+        <LinkContainer>
+          <h2>
+            <a href="onepager.pdf" target="_blank" rel="noopener noreferrer">
+              <i className="fas fa-file-alt"></i>Read our onepager
+            </a>
+          </h2>
+        </LinkContainer>
+      </div>
+      <div>
+        <LinkContainer>
+          <h2>
+            <a
+              href="xd.adobe.com/view/395518bf-7e51-4eb2-6ff2-3f1b57fdaa17-4f64/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <i className="fas fa-pencil-ruler"></i>View our interactive prototype
+            </a>
+          </h2>
+        </LinkContainer>
+      </div>
+      <div>
+        <LinkContainer>
+          <h2>
+            <Link href="/faq">
+              <a>
+                <i className="fas fa-question"></i>View our FAQ section for in depth explanations
+              </a>
+            </Link>
+          </h2>
+        </LinkContainer>
+      </div>
+      <h1>Join our Team</h1>
+      <MemberInfoGrid>{renderMemberInfo(members)}</MemberInfoGrid>
+      <h2>Currently all members of our team are volunteers.</h2>
+      <h1>Interested in joining us in accelerating climate action worldwide?</h1>
+      <h1>Contact us at contact@climateconnect.earth</h1>
+      <div>
+        While we are looking for any motivated people to join us, these skills would be especially
+        helpful:
+      </div>
+      <RequiredSkills>
+        <li>Front end development (react)</li>
+        <li>Back end development (node, postgres)</li>
+        <li>Webdesign</li>
+        <li>Marketing (social media marketing, creation of an image video)</li>
+        <li>Knowledge in the assessment of CO2-footprints of processes / knowledge in LCA</li>
+        <li>Generally experience with founding and working in a non-profit organisation</li>
+      </RequiredSkills>
+    </Container>
+  );
+}
+
+const InfoBubble = ({ data }) => (
+  <InfoBubbleContainer>
+    <Bubble>
+      <IconWrapper>
+        <i className={"fas " + data.icon}></i>
+      </IconWrapper>
+    </Bubble>
+    <h3>{data.title}</h3>
+    <InfoText>{data.text}</InfoText>
+  </InfoBubbleContainer>
+);
```

Comment:
Could probably make this it's own component, easier for testing and would cleanup this file some

Reply:
Yeah would recommend that for sure especially with it having 4 styled-components